camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.Demo.CameraToHls.Pipeline do
+  use Membrane.Pipeline
+
+  @impl true
+  def handle_init(_) do
+    children = [
+      # Captures video from the camera (raw video, depending on camera/os)
+      source: Membrane.CameraCapture,
+      # Converts pixel format to I420 (this is still a raw video, but different format)
+      converter: %Membrane.FFmpeg.SWScale.PixelFormatConverter{format: :I420},
+      # Takes raw I420 frames and converts to H264
+      # baseline profile tells encoder not to generate
+      # B-frames because browsers can't handle them
+      encoder: %Membrane.H264.FFmpeg.Encoder{profile: :baseline},
+      # Creates caps, generates metadata and adds timestamps
+      # There we need to specify framerate, based on it we
+      # generate timestamps
+      # Further we won't be able to control framerate
```

Comment:
That information about controlling framerate is:
1. probably unnecessary
1. confusing. We don't really control the framerate here, we just generate timestamps based on it

Moreover, I would rephrase it
```suggestion
      # Creates caps, generates metadata and timestamps the stream based on the framerate
```